lib/order.rb

```diff
@@ -0,0 +1,61 @@
+class Order
+
+  attr_reader :current_order, :printed_order, :menu
+
+  def initialize(restaurant)
+    @current_order = []
+    @printed_order = ""
+    @menu = Menu.new(restaurant)
+    @total = 0
+  end
+
+  def add(item)
+    @menu.csv_to_hash if @menu.menu_hash.empty?
+    fail "Input error: This item is not on the menu." unless @menu.menu_hash.key?(item)
+    @current_order.push({item => @menu.menu_hash[item]})
+  end
+
+  def remove(item)
```

Comment:
This _feels_ a little complicated for one method,, but I'm sure there are good reasons for that

Reply:
Could you use encapsulation to move some of these long operations into a private method and make the main body of the code more readable?